app/.gitignore

```diff
@@ -20,3 +20,4 @@ tmp/
 ./apache-tomcat
 apache-tomcat.zip
 config-metadata.properties
+/src/main/resources/bower_components/
```

Comment:
We should not use any bower components in CAS 6 so we should remove this entry.

Reply:
removed from .gitignore